testing/ef_tests/src/cases/kzg_verify_cell_kzg_proof_batch.rs

```diff
@@ -37,32 +37,37 @@ impl<E: EthSpec> Case for KZGVerifyCellKZGProofBatch<E> {
     fn result(&self, _case_index: usize, _fork_name: ForkName) -> Result<(), Error> {
         let parse_input = |input: &KZGVerifyCellKZGProofBatchInput| -> Result<_, Error> {
             let (cells, proofs) = parse_cells_and_proofs(&input.cells, &input.proofs)?;
+            // TODO: We are not pulling in the latest consensus-specs
+            // TODO: whereas the kzg libraries are using the new API, so
+            // TODO: we need to update this code to work for the new API.
```

Comment:
I think this has been resolved? 
We're now using the latest alpha-5 release and your changes in this PR now uses the new API.

Reply:
ah i see what you mean now, the fields in the `KZGVerifyCellKZGProofBatchInput` doesn't match the test yaml files, I'll push a commit.

Reply:
Fixed in https://github.com/sigp/lighthouse/pull/6312/commits/4295f7a7d05a07eca1a24960f42679b289363312